### DISCUSSION TITLE
dxvk_macos.verb "no such file or directory"

### DISCUSSION THREAD

**Discussion by Degetzel87** [The-Wineskin-Project/WineskinServer#288] [General]
Hello! I'm trying to get a legacy wow client running as others have done it before without any issues with excelent results.
To do this, i've had to run the dxvk_macos.verb into winetricks but get a "no such file or directory" error when running it.
I've seen that Gcenx announced that MoltenVK-DXVK was removed. Is this a possible reason?

The game runs even without it, but at unplayable fps. I've tried installing the provided dxvk DLLs included but it doesn't seem to make a difference.

Can someone shed some light into this? Thank you in advance!

**Comment by Gcenx** [comment]
Please review the projects README.md and the wiki this is all explained.
